UPGRADE-4.0.md

```diff
@@ -0,0 +1,9 @@
+UPGRADE FROM 3.x to 4.0
+=======================
+
+### DoctrineBridge and Validator
+
+ * If you were using the `StaticMethodLoader` to create `UniqueEntity`
```

Comment:
Isn't is possible to iterate over `$metadata->getConstraints()` in the `StaticMethodLoader` in order to avoid this ?
```php
foreach ($metadata->getConstraints() as $constraint) {
    if ($constraint instanceof TargetAwareConstraintInterface) {
        $constraint->target = $metadata->getClassName();
    }
}
```